angular-client/src/components/battery/battery.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="battery">
+  <div
+    class="battery-nub"
+    [ngStyle]="{
```

Comment:
why isnt this in the css file

Reply:
all this css style stuff rn is dependent on the input component variables so i have to do it this way. i looked around online for alternatives but everywhere I looked said you had to use ngStyle like so. if u know another way to do it let me know

Reply:
https://stackoverflow.com/questions/46047502/generate-dynamic-css-based-on-variables-angular here's a reference stackoverflow about it